api/Notification.json

```
@@ -910,7 +910,15 @@
               "version_added": "17"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "117",
+              "notes": "Enabled on Windows by default in Firefox 117.",
```

how about the following rewording (and should we remove "Firefox 117" because I am guessing the release version will imply that?):

```suggestion
              "notes": "Enabled by default on Windows",
```